### PR TITLE
ENG-3690 amend svc restart command

### PR DESCRIPTION
**Pull request by jeffgo10**

### CODE REVIEW

File: src/commands/svc/restart.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+  static examples = [
+    '<%= config.bin %> <%= command.id %> --all',
+    '<%= config.bin %> <%= command.id %> ext-service',
+    '<%= config.bin %> <%= command.id %> ext-service1 ext-service2'
+  ]
+
+  static flags = {
+    all: Flags.boolean({
+      description: 'Stops all enabled services in the bundle descriptor'
```
Comment (cecchisandrone) [inline src/commands/svc/restart.ts]: Stops -> Restart. Also please check verbs in other flag commands as I see we still have wrong verbs

File: src/commands/svc/restart.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    const { argv, flags } = await this.parse(Restart)
+
+    const svcService: SvcService = new SvcService(
+      process.cwd(),
+      this.config.bin
+    )
+
+    if (!flags.all && argv.length === 0) {
+      this.error(
+        'At least one service name is required. You can also use `--all` flag to stop all running enabled services',
```
Comment (cecchisandrone) [inline src/commands/svc/restart.ts]: restart

File: src/services/svc-service.ts [hunk 3]
```diff
@@ -107,6 +108,14 @@ export class SvcService {
     return this.processDockerExecution(DockerServiceType.STOP, services)
   }
 
+  public restartServices(services: string[]): Promise<ProcessExecutionResult> {
+    this.precheckEnabledServices(services)
+
+    SvcService.debug(`restarting service ${services.join(', ')}`)
+
+    return this.processDockerExecution(DockerServiceType.RESTART, services)
```
Comment (cecchisandrone) [inline src/services/svc-service.ts]: this should be `executeDockerComposeCommand`

File: src/services/svc-service.ts [hunk 4]
```diff
@@ -13,7 +13,8 @@ import { debugFactory } from './debug-factory-service'
 
 enum DockerServiceType {
```
Comment (cecchisandrone) [inline src/services/svc-service.ts]: DockerComposeCommand

File: src/commands/svc/restart.ts (hunk 1 above)
Comment (jeffgo10) [reply src/commands/svc/restart.ts]: done

File: src/commands/svc/restart.ts (hunk 2 above)
Comment (jeffgo10) [reply src/commands/svc/restart.ts]: done

File: src/services/svc-service.ts (hunk 3 above)
Comment (jeffgo10) [reply src/services/svc-service.ts]: done

File: src/services/svc-service.ts (hunk 4 above)
Comment (jeffgo10) [reply src/services/svc-service.ts]: done

File: src/commands/svc/restart.ts [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+  ]
+
+  static flags = {
+    all: Flags.boolean({
+      description: 'Restarts all enabled services in the bundle descriptor'
+    })
+  }
+
+  public async run(): Promise<void> {
+    BundleService.verifyBundleInitialized(process.cwd())
```
Comment (zonia3000) [inline src/commands/svc/restart.ts]: @jeffgo10 Please merge with develop and update this (method has been changed to `BundleService.isValidBundleProject()` without `process.cwd()`). Thanks

Comment (jeffgo10) [reply src/commands/svc/restart.ts]: done

**Review by cecchisandrone — APPROVED** [review]

**Pull request merged**